packages/ketcher-core/src/chemistry/entities/SGroup.ts

```diff
@@ -230,23 +237,25 @@ export class SGroup {
           bbb = bbb === null ? new Box2Abs(p, p) : bbb!.include(p)
         })
       })
-      bb = bb === null ? bbb : Box2Abs.union(bb, bbb!)
+      braketBox = braketBox === null ? bbb : Box2Abs.union(braketBox, bbb!)
```

Comment:
could you replace `braketBox === null` by `!braketBox` ?

Reply:
I'm not sure about such change